src/compiler/scala/tools/nsc/reporters/StoreReporter.scala

```diff
@@ -8,22 +8,23 @@ package reporters
 
 import scala.collection.mutable
 import scala.reflect.internal.util.Position
+// TODO
+//import scala.reflect.internal.Reporter
 
 /**
  * This class implements a Reporter that stores its reports in the set `infos`.
  */
 class StoreReporter extends Reporter {
   case class Info(pos: Position, msg: String, severity: Severity) {
-    override def toString() = "pos: " + pos + " " + msg + " " + severity
+    override def toString() = s"pos: $pos $msg $severity"
   }
   val infos = new mutable.LinkedHashSet[Info]
-  protected def info0(pos: Position, msg: String, severity: Severity, force: Boolean) {
+  override protected def info0(pos: Position, msg: String, severity: Severity, force: Boolean): Unit = {
```

Comment:
I think we usually don't put `override` when only implementing an abstract method.

Reply:
OK. I'll also normalize NoReporter, above. :)